tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/PlaybackTests.cs

```diff
@@ -20,7 +20,9 @@ public async void TestStartPlaybackSimple()
         {
             RecordingHandler testRecordingHandler = new RecordingHandler(Directory.GetCurrentDirectory());
             var httpContext = new DefaultHttpContext();
-            httpContext.Request.Headers["x-recording-file"] = "Test.RecordEntries/requests_with_continuation.json";
+            var body = "{\"x-recording-file\":\"Test.RecordEntries/requests_with_continuation.json\"}";
```

Comment:
Can we add a test case with non-ascii chars so we make sure everything works as expected?

Reply:
Yeah. Definitely.